spanner-ITs-against-emulator.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License..
+
+# Fail on any error
+set -e
+
+# Display commands being run
+set -x
+
+export SPANNER_EMULATOR_HOST=localhost:9010
+export TEST_PROJECT=emulator-test-project
+echo "Running the Cloud Spanner emulator: $SPANNER_EMULATOR_HOST";
+
+# Download the emulator
+EMULATOR_VERSION=0.8.0
```

Comment:
I think they will for the GA launch. I have now changed the script to use gcloud because that was the only way that I could figure out how to create an instance through the command line.